Tools/machines/perlmutter-nersc/perlmutter_warpx.profile.example

```diff
@@ -1,31 +1,14 @@
 # please set your project account
-#export proj="<yourProject>_g"  # change me
+export proj="<yourProject>_g"  # change me
 
-# required dependencies
-module load cmake/3.22.0
+# load E4S & Spack
+module load e4s/22.05
```

Comment:
@eugeneswalker told me there is now 22.11 on NERSC: https://dashboard.e4s.io/

Can be loaded with
```
. /global/cfs/cdirs/m3896/shared/ParaTools/E4S/22.11/PrgEnv-gnu/module-use.sh 
```

Reply:
I think he meant
```
. /global/cfs/cdirs/m3896/shared/ParaTools/E4S/22.11/PrgEnv-gnu/start-spack.sh
```

Reply:
Actually, I did mean `module-use.sh` because we typically expose via modules. We use `start-spack.sh` internally to modify installations, but because end-user don't have write permission to the directories, usually people use `module-use.sh` to access the packages. Do you prefer to have the Spack instance sourced?

Reply:
Yes, what I am doing here is build my own Spack env on top of NERSC's base packages. Thus I need a `spack` command :)

Reply:
Got it :)
```console
. /global/cfs/cdirs/m3896/shared/ParaTools/E4S/22.11/PrgEnv-gnu/module-use.sh
. /global/cfs/cdirs/m3896/shared/ParaTools/E4S/22.11/PrgEnv-gnu/spack/share/spack/setup-env.sh
```